website/docs/r/org.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org"
+sidebar_current: "docs-vcd-resource-org"
+description: |-
+Provides a vCloud Director Organization resource. This can be used to create and delete an organization.
+---
+
+# vcd\_org
+
+Provides a vCloud Director Org resource. This can be used to create and delete an organization.
+
+Supported in provider *v2.0+*
+
+## Example Usage
+
+```
+resource "vcd_org" "my-org" {
+  name             = "my-org"
+  full_name        = "My organization"
+  description      = "The pride of my work"
+  is_enabled       = "true"
+  delete_force     = "true"
+  delete_recursive = "true"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Org name
+* `full_name` - (Required) Org full name
+* `delete_recursive` - (Required) - pass `delete_recursive`=true as query parameter to remove an organization or VDC and any objects it contains that are in a state that normally allows removal.
```

Comment:
Strange code quoting. What about this (include the true/false assignment in the code quote):
`delete_recursive=true`